build.rs

```diff
@@ -84,6 +86,26 @@ fn main() {
     if linux_time_bits64 {
         set_cfg("linux_time_bits64");
     }
+    println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64");
+    match env::var("RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64") {
```

Comment:
I think we may want to use `RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS` and then have it be set to `=32` or `=64`. Reason being, I think in 1.0 we will default to 64-bit but need to provide an environment variable to use 32.

Reply:
Also, doesn't this need to set `linux_time_bits64` as well?